components/hidden_data_box/wikis/starcraft2/hidden_data_box_custom.lua

```diff
@@ -0,0 +1,82 @@
+---
+-- @Liquipedia
+-- wiki=starcraft2
+-- page=Module:HiddenDataBox/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Variables = require('Module:Variables')
+local BasicHDB = require('Module:HiddenDataBox')
+local CustomHDB = {}
+
+function CustomHDB.run(args)
+	BasicHDB.addCustomVariables = CustomHDB.addCustomVariables
+	return BasicHDB.run(args)
+end
+
+function CustomHDB:addCustomVariables(args, queryResult)
+	--legacy variables
+	Variables.varDefine('tournament_tier', Variables.varDefault('tournament_liquipediatier', ''))
+	Variables.varDefine('tournament_tiertype', Variables.varDefault('tournament_liquipediatiertype', ''))
+	Variables.varDefine('tournament_date', Variables.varDefault('tournament_enddate', ''))
+	Variables.varDefine('tournament_edate', Variables.varDefault('tournament_enddate', ''))
+	Variables.varDefine('tournament_sdate', Variables.varDefault('tournament_startdate', ''))
+	Variables.varDefine('tournament_ticker_name', Variables.varDefault('tournament_tickername', ''))
```

Comment:
A good number of these are legacy, please mark them as such.

Reply:
see line 20